R/create_output_type_item.R

```diff
@@ -190,9 +190,14 @@ create_output_type_point <- function(output_type = c("mean", "median"),
 #'   value_type = "double"
 #' )
 #' create_output_type_sample(
-#'   required = 1:10, optional = 11:15,
-#'   value_type = "double"
-#' )
+#'  is_required = TRUE,
+#'  output_type_id_type = "integer",
+#'  min_samples_per_task = 70L, max_samples_per_task = 100L,
+#'  value_type = "double",
+#'  value_minimum = 0L,
+#'  value_maximum = 1L,
```

Comment:
related to a question I asked on the schemas PR: if someone calling this function just provided `0` and `1` instead of `0L` and `1L` would that be ok?

Reply:
Link to related answer here: https://github.com/Infectious-Disease-Modeling-Hubs/schemas/pull/72/files/a7f81a6c7c4135601e1ce506659a2b25657b265d#r1592004920

Happy to change if you prefer, just let me know

Reply:
Actually, went ahead and changed it to avoid confusion in 810ba0b5b4f3ab9d695083a70d4bbaf8dbfbc464

Reply:
ok, just to clarify, i just wanted to make sure it would work since all the examples i saw had integer type specified.